lib/elixir/pages/Writing Documentation.md

```diff
@@ -51,6 +51,28 @@ The compiler will infer this argument as `map`. Sometimes the inference will be
       size
     end
 
+## Documentation metadata
+
+Elixir allows developers to attach random metadata to the documentation too. This is done by using the relevant attribute, such as `@moduledoc`, `@typedoc` and `@doc` and passing a keyword list where the key is the metadata key and value is the metadata value. A commonly used metadata is `:since`, which annotates which version that particular module, function, type or callback was added. For instance, in the example above we could have added:
+
+    defmodule MyApp.Hello do
```

Comment:
What do you think about giving a more complete example?

```elixir
defmodule MyApp.HelloWorld do
  @moduledoc "Says hello to the world"
  @moduledoc since: "1.0.0"

  @doc "Prints hello world"
  @doc since: "1.3.0"
  def hello_world, do: IO.puts("Hello world!")
end
```

Reply:
There is an example at the top of the file that I am referring to but you can't see it in a diff. Isn't that enough?